samples/sample_plugins/vpp_plugin/include/mfx_multi_vpp.h

```diff
@@ -32,32 +32,39 @@ class MFXVideoMultiVPP : public MFXVideoVPP
     virtual ~MFXVideoMultiVPP(void) { Close(); }
 
     // topology methods
-    virtual mfxStatus QueryIOSurf(mfxVideoParam *par, mfxFrameAllocRequest request[2], mfxVideoParam *par1 = NULL, mfxVideoParam *par2 = NULL)
-    { par1; par2; return MFXVideoVPP_QueryIOSurf(m_session, par, request); }
+    virtual mfxStatus QueryIOSurf(mfxVideoParam *par, mfxFrameAllocRequest *request) override { return QueryIOSurfMulti(par, request); }
```

Comment:
I will review this part once more, because still didn't get what was the issue. I hope could overcome the issue in some simpler way

Reply:
Issue was a hides overloaded virtual function, because parent methods had more arguments than child methods.

Reply:
They wanted to have QueryIOSurf which accepts 3 parameters while original one accepted only 2.